jobbergate-core/README.rst

```diff
@@ -0,0 +1,15 @@
+=================
+ Jobbergate Core
+=================
+
+Jobbergate-core is a sub-project that contains the key components and logic that is shared among all other sub-projects (CLI, API, and Agent). Additionally, jobbergate-core exists to support custom automation built on top of Jobbergate.
+
+License
+-------
+* `MIT <LICENSE>`_
+
+
+Copyright
```

Comment:
We should not call out the copyright her since we are already referencing the license which contains this information. Perhaps a link to CHANGELOG would be more appropriate.

Reply:
It is just following the patter of the project. since it is included in the README for CLI and API. Should I remove all of them?

Reply:
No, let's leave it like this for now. Something to think about for the future.